packages/mock-account-provider/app/lib/utils.ts

```diff
@@ -37,3 +38,24 @@ export function parseQueryString(query: string) {
     }
   }
 }
+
+type OpenPaymentsJWK = JsonWebKey & {
+  alg: 'EdDSA'
+  kid: string
+}
+
+export const generateJwk = ({ keyId }: { keyId: string }): OpenPaymentsJWK => {
+  const jwk = createPublicKey(generateKeyPairSync('ed25519').privateKey).export(
+    {
+      format: 'jwk'
+    }
+  )
+
+  return {
+    alg: 'EdDSA',
+    kid: keyId,
+    kty: jwk.kty,
+    crv: jwk.crv,
+    x: jwk.x
+  }
```

Comment:
I like having this explicit instead of a spread because any additional or any fewer keys would cause the Open Payment validation to fail if asking for `{{paymentPointer}}/jwks.json`